pkg/sdn/plugin/proxy.go

```diff
@@ -78,7 +79,7 @@ func (proxy *ovsProxyPlugin) watchEgressNetworkPolicies() {
 			policy.Spec.Egress = nil
 		}
 		proxy.updateNetworkPolicy(*policy)
-		// FIXME: poke the endpoint-syncer somehow...
+		proxy.OnEndpointsUpdate(proxy.lastEndpoints)
```

Comment:
Hm... yeah, it could mean that if you restart the node and then bad timing occurs, you could lose all your iptables rules briefly